include/solver_enums.h

```diff
@@ -25,11 +25,13 @@ enum SolverEnum
   CVC4,
   MSAT,
   YICES2,
+  Z3,
   // have separate enum for solver wrapped by LoggingSolver with Shadow DAG
   BTOR_LOGGING,
   CVC4_LOGGING,
   MSAT_LOGGING,
   YICES2_LOGGING,
+  Z3_LOGGING,
```

Comment:
Just FYI, in the latest smt-switch we don't have the separate *_LOGGING enums. This shouldn't be a problem, you'll just need to resolve this when merging in the upstream changes (from the main smt-switch master branch)